playground/README.md

```diff
@@ -0,0 +1,8 @@
+# LWC playground
+
+The simplest LWC setup for simple experiments and bug reproductions.
+
+```sh
+$ npm run dev       # Get app server running
+$ npm run build     # Build app in production mode
+```
```

Comment:
I'd add some instructions for running in prod mode, e.g. `npx serve` or something, even if we don't have `serve` installed locally.